src/modules/cardTypes/cardTypes.service.ts

```diff
@@ -165,11 +173,18 @@ export class CardTypesService {
         ),
         tokens,
       );
-      return await this.cardTypesRepository.save(foundCardTpyes);
+  
+      const response = {
+        message: `Card type updated successfully.`,
```

Comment:
can we move this string into the strings file?

Reply:
![image](https://github.com/user-attachments/assets/f415e6a0-35af-4a64-b70c-9757f1ed10f6)
Can I add a constant string or is it better to use one?

Reply:
you can add a constants strings

Reply:
Ready